scheduler/scheduler.go

```diff
@@ -0,0 +1,77 @@
+package scheduler
+
+import (
+	"io/ioutil"
+	"strings"
+	"time"
+)
+
+var zoneDirs = []string{
+	// Update path according to your OS
+	"/usr/share/zoneinfo/",
+	"/usr/share/lib/zoneinfo/",
+	"/usr/lib/locale/TZ/",
+}
+
+var zoneDir string
+
+// Scheduler :
+type Scheduler struct {
+}
+
+//NewScheduler : constructor
+func (scheduler *Scheduler) NewScheduler() *Scheduler {
+	s := Scheduler{}
+	go s.Start()
+	return &s
+}
+
+// Start : start and check every hour for scheduled summaries
+func (scheduler *Scheduler) Start() {
+
+	t := time.NewTicker(time.Hour * 1)
```

Comment:
Yo this is perfect, so it just calls `Do()` every hour then increments? Is that what `<-t.C` does?

See my previous comment, this makes things much simpler 😄 A more descriptive function name than `Do()` would help too